packages/workflow/src/Workflow.ts

```diff
@@ -1327,12 +1330,22 @@ export class Workflow {
 				additionalData,
 				executionData,
 				mode,
+				closeFunctions,
 				abortSignal,
 			);
 			const data =
 				nodeType instanceof Node
 					? await nodeType.execute(context)
 					: await nodeType.execute.call(context);
+
+			closeFunctions.forEach(async (closeFunction) => {
+				try {
+					await closeFunction();
+				} catch (error) {
+					throw new NodeOperationError(node, error as Error);
```

Comment:
Uh looking through my code. It should still try to run the other closeFunctions before throwing the error.

Reply:
@janober Pushed an update to run all closeFunction before throwing an error, lmk if it makes sense to you